lib/Wikia/src/Service/User/Attributes/UserAttributes.php

```diff
@@ -126,7 +117,7 @@ public function deleteAttribute( $userId, Attribute $attribute ) {
 
 	private function attributeNotSetForUser( $userId, Attribute $attribute ) {
 		$this->loadAttributes( $userId );
-		return empty( $this->attributes[$userId][$attribute->getName()] );
```

Comment:
This was actually incorrect since it would return true if the *value* of `$this->attributes[$userId][$attribute->getName()];` was falsey, we only want to know if that key exists or not.